packages/rollup-plugin/src/index.ts

```diff
@@ -210,26 +229,27 @@ export function stylableRollupPlugin({
             };
 
             for (const moduleId of this.getModuleIds()) {
-                if (moduleId.endsWith(ST_CSS)) {
-                    modules.push({ depth: calcDepth(moduleId, context, [], cache), moduleId });
+                const { isStFile, isLoadableCssFile, path } = getLoadableModuleData(moduleId);
+                if (isStFile || isLoadableCssFile) {
+                    modules.push({ depth: calcDepth(moduleId, context, [], cache), path });
```

Comment:
why do we calc depth for the css? I'm not sure that algorithm knows how to handle it.

Reply:
it doesn't mean anything. will always be 0